### PR TITLE
LPS-86596, Custom Filter Widget Language Key Edits

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LPS-86596, wordsmithing the language keys.

### CODE REVIEW

**Comment by rbohl** [comment]
cc @bryanengler @lipusz, if these look good to you guys I can build the default copies and send to bchan. Let me know.

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -85,17 +85,17 @@ maximum-number-of-related-queries=Maximum Number of Related Queries
...
 no-widgets-were-searched=No widgets were searched.
 occur=Occur
-occur-help=The occurrence type of the clause to add when this query is contributed. Behavior for each Occur type is defined by search engines.
+occur-help=Sets the occurrence type for the query being contributed to the search. The behavior of each type is defined by the search engine.
 pagination-delta=Pagination Delta
 pagination-delta-parameter-name=Pagination Delta Parameter Name
 pagination-start-parameter-name=Pagination Start Parameter Name
 parent-query-name=Parent Query Name
-parent-query-name-help=The Query Name as configured at the parent Custom Filter widget, only when this widget is intended to contribute a child clause to a Bool query. Should be left blank otherwise.
+parent-query-name-help=The Query Name configured in the parent Custom Filter widget. Leave this blank unless contributing a child clause to a Bool query.
```
Comment (lipusz) [inline modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: I'd use the term `Boolean Query` instead as it's more standard.

**Comment by lipusz** [comment]
Added a note, I'm fine with the rest on my side. Thanks @rbohl ! :+1:

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties (hunk 1 above)
Comment (rbohl) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: @lipusz we refer to it in the configuration UI as a "Boo" though, so I think we should be consistent in the help text. If we wanted to rename it to Boolean everywhere it appears in the UI, that's probably a good idea, but I'm not sure if it'll get in on time. @bryanengler?

Comment (lipusz) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: Makes sense @rbohl

Comment (lipusz) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: >Boo

would be the ultimate choice I agree :rofl: :stuck_out_tongue_closed_eyes:

**Comment by rbohl** [comment]
Forwarded: https://github.com/brianchandotcom/liferay-portal/pull/71811

**Pull request closed without merging**